File: VisualBasicAssignment/Module1.vb
Module Module1
    Public con As New System.Data.SqlClient.SqlConnection
    Public editid As String
    Public detailid As String

    Public Function openconnect() As Boolean
        Dim connString As String = "Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=C:\Users\user\Documents\GitHub\xvb-food-ordering-system\VisualBasicAssignment\KZDB.mdf;Integrated Security=TrueData Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=C:\Users\user\Documents\GitHub\xvb-food-ordering-system\VisualBasicAssignment\KZDB.mdf;Integrated Security=True"
        Try
            con.ConnectionString = connString
            con.Open()
            Return True
        Catch ex As Exception
            Return False
        End Try
    End Function

    Public Sub closeconnect()
        con.Close()
    End Sub




End Module

File: VisualBasicAssignment/Module1.cs
using System;

namespace VisualBasicAssignment
{
    static class Module1
    {
        public static System.Data.SqlClient.SqlConnection con = new System.Data.SqlClient.SqlConnection();
        public static string editid;
        public static string detailid;

        public static bool openconnect()
        {
            string connString = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\FoodShop.mdf;Integrated Security=True";
            try
            {
                con.ConnectionString = connString;
                con.Open();
                return true;
            }
            catch (Exception ex)
            {
                return false;
            }
        }

        public static void closeconnect()
        {
            con.Close();
        }
    }
}